pkg/config/util.go

```diff
@@ -22,5 +21,5 @@ func ToJson(cfg Config) ([]byte, error) {
 		return nil, err
 	}
 	// there is no easy way to convert yaml to json using gopkg.in/yaml.v2
-	return ghodss_yaml.YAMLToJSON(yamlBytes)
+	return yaml.YAMLToJSON(yamlBytes)
 }
```

Comment:
Sounds like if we were to switch from `yaml` tags to `json` tags everywhere in config we'd be able to get rid of this entire package and more importantly this double serialization right?

If that's the case could you please open an issue to do this. Thanks!

Reply:
It's done :) !